tests/Type/Php/PregMatchParameterOutTypeExtensionTest.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types = 1);
+
+namespace TheCodingMachine\Safe\PHPStan\Type\Php;
+
+use PHPStan\Testing\TypeInferenceTestCase;
+
+class PregMatchParameterOutTypeExtensionTest extends TypeInferenceTestCase
+{
+
```

Comment:
This should be fixed by php-cs-fixer

Reply:
Looks like `tests/` aren't covered by the formatter - I'll manually fix for now, and then cover tests in a follow-up